worlds/lufia2ac/__init__.py

```diff
@@ -0,0 +1,351 @@
+        self.boss = self.multiworld.boss[self.player]
+        self.capsule_cravings_jp_style = self.multiworld.capsule_cravings_jp_style[self.player].value
+        self.capsule_monsters_available = self.multiworld.capsule_monsters_available[self.player].value
+        self.capsule_starting_form = self.multiworld.capsule_starting_form[self.player]
+        self.capsule_starting_level = self.multiworld.capsule_starting_level[self.player]
+        self.crowded_floor_chance = self.multiworld.crowded_floor_chance[self.player].value
+        self.death_link = self.multiworld.death_link[self.player].value
+        self.final_floor = self.multiworld.final_floor[self.player].value
+        self.gear_variety_after_b9 = self.multiworld.gear_variety_after_b9[self.player].value
+        self.goal = self.multiworld.goal[self.player].value
+        self.healing_floor_chance = self.multiworld.healing_floor_chance[self.player].value
+        self.initial_floor = self.multiworld.initial_floor[self.player].value
+        self.iris_floor_chance = self.multiworld.iris_floor_chance[self.player].value
+        self.iris_treasures_required = self.multiworld.iris_treasures_required[self.player].value
+        self.master_hp = self.multiworld.master_hp[self.player].value
+        self.party_members_available = self.multiworld.party_members_available[self.player].value
+        self.party_starting_level = self.multiworld.party_starting_level[self.player]
+        self.run_speed = self.multiworld.run_speed[self.player].value
+        self.starting_capsule = self.multiworld.starting_capsule[self.player].value
+        self.starting_party = self.multiworld.starting_party[self.player]
+
+        if self.capsule_starting_level.value == CapsuleStartingLevel.special_range_names["party_starting_level"]:
+            self.capsule_starting_level.value = self.party_starting_level.value
+        if self.initial_floor >= self.final_floor:
+            self.initial_floor = self.final_floor - 1
+        if self.master_hp == MasterHp.special_range_names["scale"]:
+            self.master_hp = MasterHp.scale(self.final_floor)
+        if self.party_members_available == PartyMembersAvailable.option_shuffled_into_multiworld:
+            self.starting_party.value = 0x00000001
+
+    def create_regions(self) -> None:
+        menu = Region("Menu", RegionType.Generic, "Menu", self.player, self.multiworld)
+        menu.exits.append(Entrance(self.player, "AncientDungeonEntrance", menu))
+        self.multiworld.regions.append(menu)
+
+        ancient_dungeon = Region("AncientDungeon", RegionType.Generic, "Ancient Dungeon", self.player, self.multiworld)
+        ancient_dungeon.exits.append(Entrance(self.player, "FinalFloorEntrance", menu))
+        item_count: int = self.blue_chest_count
+        if self.capsule_monsters_available == CapsuleMonstersAvailable.option_shuffled_into_multiworld:
+            item_count += len(self.item_name_groups["Capsule monsters"])
```

Comment:
these options seem to increase the number of blue chests beyond the number chosen by the user, but this doesn't seem to be communicated in the option descriptions